core/src/software/aws/toolkits/core/utils/PathUtils.kt

```diff
@@ -11,7 +11,7 @@ import java.nio.file.attribute.FileTime
 
 fun Path.inputStream(): InputStream = Files.newInputStream(this)
 fun Path.exists() = Files.exists(this)
-fun Path.delete() = Files.delete(this)
+fun Path.delete() = Files.deleteIfExists(this)
```

Comment:
I'd probably change the alias name too to `Path.deleteIfExists`

Reply:
makes sense